extras/sphinxtogithub/sphinxtogithub.py

```diff
@@ -97,7 +97,7 @@ class DirectoryHandler:
     def __init__(self, name, root, renamer):
         self.name = name
         self.new_name = name[1:]
-        self.root = root + os.sep
+        self.root = str(root)
```

Comment:
I fixed this up post-merge to use `str(root) + os.sep` because the code outside of the constructor does things like `path = directory.replace(self.root, "", 1)` (Cf. `def relative_path()`) which currently relies on the trailing `/` being there so that it replaces all the way through the `/`.

It doesn't seem to make a difference in cola's usage, but it doesn't hurt to fix this up in case we end up going down that code path in the future so I tweaked this to keep `+ os.sep` at the end.